### ISSUE TITLE
"repomd.xml signature could not be verified for google-cloud-sdk" bootstrap failures on redhat nodes

### ISSUE THREAD

**Issue by wadells** [gravitational/robotest#282]
All robotest runs on redhat started failing today.  The bootstrap script fails on:

https://github.com/gravitational/robotest/blob/7a452649633d9668ecc599be87f6fb2810037eba/assets/terraform/gce/bootstrap/centos.sh#L96

with the following signature:

```
[redhat@walt-rhel-debug-node-0 ~]$ sudo yum --disablerepo=epel -y update ca-certificates
google-cloud-sdk/signature                                                                                                                                             |  844 B  00:00:00     
Retrieving key from https://packages.cloud.google.com/yum/doc/yum-key.gpg
Retrieving key from https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
google-cloud-sdk/signature                                                                                                                                             | 1.4 kB  00:00:00 !!! 
https://packages.cloud.google.com/yum/repos/cloud-sdk-el7-x86_64/repodata/repomd.xml: [Errno -1] repomd.xml signature could not be verified for google-cloud-sdk
Trying other mirror.


 One of the configured repositories failed (Google Cloud SDK),
 and yum doesn't have enough cached data to continue. At this point the only
 safe thing yum can do is fail. There are a few ways to work "fix" this:

     1. Contact the upstream for the repository and get them to fix the problem.

     2. Reconfigure the baseurl/etc. for the repository, to point to a working
        upstream. This is most often useful if you are using a newer
        distribution release than is supported by the repository (and the
        packages for the previous distribution release still work).

     3. Run the command with the repository temporarily disabled
            yum --disablerepo=google-cloud-sdk ...

     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:

            yum-config-manager --disable google-cloud-sdk
        or
            subscription-manager repos --disable=google-cloud-sdk

     5. Configure the failing repository to be skipped, if it is unavailable.
        Note that yum will try to contact the repo. when it runs most commands,
        so will have to try and fail each time (and thus. yum will be be much
        slower). If it is a very temporary problem though, this is often a nice
        compromise:

            yum-config-manager --save --setopt=google-cloud-sdk.skip_if_unavailable=true

failure: repodata/repomd.xml from google-cloud-sdk: [Errno 256] No more mirrors to try.
https://packages.cloud.google.com/yum/repos/cloud-sdk-el7-x86_64/repodata/repomd.xml: [Errno -1] repomd.xml signature could not be verified for google-cloud-sdk
```

**Comment by UnderGreen** [comment]
Hmm, I feel like I saw this error already and then it after some time disappeared. Maybe a temporary problem with google cloud RH repo or with the image.

**Comment by wadells** [comment]
This update has been a re-occuring source of pain.  See also: https://github.com/gravitational/robotest/issues/214

**Comment by wadells** [comment]
Google just updated the key from the failing:

```
Retrieving key from https://packages.cloud.google.com/yum/doc/yum-key.gpg                                                                                                                     
Importing GPG key 0xA7317B0F:                                                                                                                                                                 
 Userid     : "Google Cloud Packages Automatic Signing Key <gc-team@google.com>"                                                                                                              
 Fingerprint: d0bc 747f d8ca f711 7500 d6fa 3746 c208 a731 7b0f                                                                                                                               
 From       : https://packages.cloud.google.com/yum/doc/yum-key.gpg
```

To the successful:

```
Retrieving key from https://packages.cloud.google.com/yum/doc/yum-key.gpg
Importing GPG key 0x307EA071:
 Userid     : "Rapture Automatic Signing Key (cloud-rapture-signing-key-2021-03-01-08_01_09.pub)"
 Fingerprint: 7f92 e05b 3109 3bef 5a3c 2d38 feea 9169 307e a071
 From       : https://packages.cloud.google.com/yum/doc/yum-key.gpg
```

**Comment by wadells** [comment]
Merged in 2.3 line of development, and released as a 2.2.2 hotfix. I don't plan to backport any further than that.
